statick_tool/exceptions.py

```diff
@@ -78,9 +78,11 @@ def filter_file_exceptions_early(self, package, file_list):
                 if exception["tools"] == 'all':
                     for pattern in exception["globs"]:
                         # Hack to avoid exceptions for everything on Travis CI.
-                        if pattern == '*/build/*':
-                            filename.lstrip('/home/travis/build/')
-                        if fnmatch.fnmatch(filename, pattern):
+                        fname = filename
```

Comment:
This fixes the `*/build/*` exception glob from being able to hide all the issues when building on Travis CI, which uses `/home/travis/build/` as the build directory. Identified in #65.